### PR TITLE
Feature add bios for blog authors

### PR DESCRIPTION
**Pull request by marsolekk**

### CODE REVIEW

File: blog/static/blog/scripts/showAuthorBio.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+    }
+});
+
+document.body.addEventListener("mouseover", function(evt) {
+    if (evt.target.className === 'author-name') {
+        var targetAuthor = evt.target.children[0];
+        var lengthOfAuthorBio = targetAuthor.getElementsByClassName("text")[0].innerText.trim().length;
+
+        if (lengthOfAuthorBio !== 0) {
```
Comment (kbeker) [inline blog/static/blog/scripts/showAuthorBio.js:24]: Please just change it to 
```javascript
if (lengthOfAuthorBio > 0)
```

File: blog/templates/blog/blog_posts_list.haml [hunk 2]
```diff
@@ -24,7 +24,21 @@
...
+                                            .bio-author
+                                                .bio-img
+                                                    {% thumbnail article.specific.author.front_image "400x400" as image %}
+                                                        %img.bio-img.lazyload{:data-src => "{{ image.url }}", :alt => "{{ article.specific.author.full_name }}"}
+                                                    {% endthumbnail %}
+                                                .bio-name
+                                                    {{ article.specific.author.full_name }}
+                                            .bio-text
+                                                .text
+                                                    {{ article.specific.author.bio|urlize }}
```
Comment (kbeker) [inline blog/templates/blog/blog_posts_list.haml:41]: I see that this is duplicated code with same thing in `blog/templates/blog/blog_posts_list_partials/main_article.haml`. This is perfect example to extract it to separate template and inject this template in this two places. Please change it

Comment (kbeker) [reply blog/templates/blog/blog_posts_list.haml:41]: I see that it is even in more than 2 places :) This is more than perfect example to extract it to separate template

**Review by kbeker — CHANGES_REQUESTED** [review]
Please follow DRY methodology because it makes code cleaner and easier to maintenance.

File: blog/static/blog/scripts/showAuthorBio.js (hunk 1 above)
Comment (marsolekk) [reply blog/static/blog/scripts/showAuthorBio.js:24]: Done

File: blog/templates/blog/blog_posts_list.haml (hunk 2 above)
Comment (marsolekk) [reply blog/templates/blog/blog_posts_list.haml]: Done

**Review by kbeker — APPROVED** [review]

**Pull request merged**